locations/method_locations_test.go

```diff
@@ -117,3 +117,36 @@ func TestMethodSignature(t *testing.T) {
 		}
 	}
 }
+
+func TestMethodOption(t *testing.T) {
+	f := parse(t, `
+		import "google/api/client.proto";
+		service Library {
+		  rpc GetBook(GetBookRequest) returns (Book) {
+		    option deprecated = true;
+		  }
+		  rpc WriteBook(NewBook) returns (Book) {}
+		}
+		message GetBookRequest{}
+		message Book {}
+		message NewBook {}
+	`)
+
+	for _, test := range []struct {
+		name        string
+		methodIdx   int
+		fieldNumber int
```

Comment:
Since the `fieldNumber` is the same for both test cases, no need for it to be a field in this struct.